docs/sources/mimir/references/architecture/components/alertmanager.md

```diff
@@ -218,8 +218,10 @@ It's rare to find cases of disagreement in a tenant configuration, as most tenan
 
 1. After identifying and fixing all affected tenant configurations, check the Mimir server logs again to make sure you haven't missed any tenant configurations.
 
-2. You can now enable UTF-8 by setting [`utf8-strict-mode-enabled`]({{< relref "./../../../configure/configuration-parameters#alertmanager" >}}) to `true`.
+2. You can now enable UTF-8 by setting [`utf8-strict-mode-enabled`]({{< relref "./../../../configure/configuration-parameters#alertmanager" >}}) to `true`. You will need to restart Mimir for the changes to take effect.
 
-3. Any incompatible tenant configurations will fail to load. You can identify if any tenant configurations are failing to load by searching the Mimir server logs for lines containing `error applying config`, or querying the `alertmanager_config_last_reload_successful` gauage for `0`.
+3. To confirm UTF-8 is enabled, search for `Starting Alertmanager in UTF-8 strict mode` in Mimir server logs.
 
-4. You can disable debug-level logging.
+4. Any incompatible tenant configurations will fail to load. You can identify if any tenant configurations are failing to load by searching the Mimir server logs for lines containing `error applying config`, or querying the `alertmanager_config_last_reload_successful` gauage for `0`.
+
+5. You can disable debug-level logging.
```

Comment:
This feels like a new procedure rather than a final step in the previous procedure. Are you planning to add instructions for how to disable debug-level logging? At the very least, I would remove this sentence from the ordered list so it doesn't appear like a step in the task.

Reply:
I haven't changed this line. The reason it's mentioned here though is at the start of the guide it asks the user to enable debug-level logging.